coremltools/converters/mil/mil/ops/defs/tensor_operation.py

```diff
@@ -516,19 +530,19 @@ def type_inference(self):
 class tile(Operation):
     """
     Returns a new tensor by replicating input ``x`` multiples times.
-    The ``i``th dimention of ``x`` will be replicated ``reps[i]`` times.
+    The ``i`` dimention of ``x`` will be replicated ``reps[i]`` times.
 
     Parameters
     ----------
     x: tensor<*?, T> (Required)
         * Input tensor.
     reps: tensor<[rank(x)], int32> (Required)
-        * A 1D tensor with length ``rank(x)`` which indicates number to replicate the input along each dimension.
+        * A 1-D tensor with length ``rank(x)``, which indicates the number to replicate the input along each dimension.
 
     Returns
     -------
     tensor<*?, T>:
-        * An Nd tensor with same type as the input.
+        * An n-D tensor with same type as the input.
```

Comment:
revert the change. Nd is a common lingo (e.g. `scatter_nd`)

Reply:
OK. (I've seen "n-D tensor" and "N-D tensor" but not "Nd tensor" until now. Note that I have been trying to consistently use "1-D", "2-D", etc.).

Reply:
OK, after a bit of discussion, decided to go with "n-D" to be consistent with "1-D", "2-D", etc.